lib/LedgerSMB/Admin/Command/user.pm

```diff
@@ -0,0 +1,582 @@
+package LedgerSMB::Admin::Command::user;
+
+=head1 NAME
+
+LedgerSMB::Admin::Command::user - ledgersmb-admin 'user' command
+
+=cut
+
+use strict;
+use warnings;
+
+use Getopt::Long qw(GetOptionsFromArray);
+use LedgerSMB::Admin::Command;
+use LedgerSMB::App_State;
+use LedgerSMB::Company;
+use LedgerSMB::Database;
+use LedgerSMB::DBObject::User;
+use LedgerSMB::Entity::Person::Employee;
+use LedgerSMB::Entity::User;
+use LedgerSMB::PGDate;
+
+use Moose;
+extends 'LedgerSMB::Admin::Command';
+use namespace::autoclean;
+
+use Feature::Compat::Try;
+
+# LedgerSMB database interface
+has db => (is => 'rw');
+has super_dbh => (is => 'rw',
+                  predicate => '_has_super_dbh');
+has admin_dbh => (is => 'rw', lazy => 1,
+                  builder => '_build_admin_dbh',
+                  clearer => '_clear_admin_dbh',
+                  predicate => '_has_admin_dbh',
```

Comment:
The builder isn't defined.Do you really eed the admin_dbh, or isn't a remnant of an early version?

Reply:
Remnant. Removed